tests/system/providers/google/cloud/bigquery/example_bigquery_transfer.py

```diff
@@ -38,7 +38,7 @@
 
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
```

Comment:
I think you want to change this line.
```suggestion
ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID", "default")
```
you have more places where you miss to add default